### DISCUSSION TITLE
so version

### DISCUSSION THREAD

**Discussion by Jackson-Pollock** [Security-Onion-Solutions/securityonion#9733] [General] [answered]
How do I determine the version of the installed security onion? on terminal, not one that is on so-soc ui.

**Comment by presianbg** [comment, marked as answer]
Hi,

Run `sudo docker ps -a` and you will get what version every container is:

![image](https://user-images.githubusercontent.com/2163486/217181502-627f94b8-0bac-4364-8341-218480237a41.png)

Cheers,
PY
